_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Podman CVMFS integration
+layout: default
+project: CernVM-FS
+year: 2020
+organization:
+ - CERN
+---
+
+## Description
+
+[CernVM-FS (CVMFS)][cvmfs] is a globally-distributed filesystem used to lazily distribute software in different server farm.
+
+[Podman][podman] is an utility to run and use containers.
+It provides the same command line interface than Docker but it runs without the need of a privileged daemon.
+These two characteristics make it extremely interesting for workload used in scientific data-centers.
+
+It has been shown that only a small portion of all the files in a container images is necessary to run the image itself.
+This is even more accentuated in scientific container images since they usually include big binaries, not always used by the specific task.
```

Comment:
big binaries --> complex software stacks comprising hundreds of thousands of files